xml/System/DateTime.xml

```diff
@@ -4812,7 +4812,8 @@ juillet 2009
         <summary>Converts the string representation of a date and time to its <see cref="T:System.DateTime" /> equivalent.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[  
-  
+
+<a name="parse-remarks"></a>  
```

Comment:
since this is not really broken, I'd prefer that we file a bug for the build report since the current link experience is better.